tools/make_patch.c

```diff
@@ -647,7 +647,7 @@ void interpret_command(char *command, const struct symbol *lastFoundSymbol, cons
 		int value = get_constant(argv[0], argv[1]);
 		if (value == -1) return;
 
-		fprintf(output, isupper(command[0]) ? "%X": "%x", value);
+		fprintf(output, isupper(command[0]) ? "%X 00": "%x 00", value);
```

Comment:
Use `%02X` and `%02x` to ensure it has a leading 0 if the number is lower than 16. I'd implement the second byte as `value >> 16`. Probably useless but doesn't hurt.

Reply:
Partially resolved in https://github.com/pret/pokecrystal/pull/882/commits/3a78bcc8073928f2675d66b38124b90b6796ffca... Will need to test the `value >> 8` soon.

Reply:
Should be fully resolved in https://github.com/pret/pokecrystal/pull/882/commits/7d4a19e7c996e4bdbae3bebbefc778a35db4e35a